lib/puppet/resource/type.rb

```diff
@@ -351,6 +390,13 @@ def set_argument_types(name_to_type_hash)
     end
   end
 
+  # Instances of this class are never capability resources, but they end up
```

Comment:
Should document what the method returns - suggest "Returns boolean true if an instance of this type is a capability. This implementation always returns false. This "duck-typing" interface is shared among other classes and makes it easier to detect capabilities when they are intermixed with non capability instances." (my interpretation of the comment)